### ISSUE TITLE
How to move the Graph UI to the side of rendering window? 

### ISSUE THREAD

**Issue by swordigo1995** [NVIDIAGameWorks/Falcor#399]
Currently, in Mogwai, the Graph UI overlaps the rendering windows by default. However, this can become a usability issue, especially when working with low-resolution windows, as the Graph UI covers most of the rendering area. This makes it challenging to observe rendering results while adjusting scene parameters.

Is it possible to move the Graph UI alongside the rendering window instead of overlapping it? Any tips on how to implement this?
@skallweitNV

**Comment by skallweitNV** [comment]
Does `F2` help to hide/show the UI?

**Comment by swordigo1995** [comment]
Yeah. But I cannot adjust scene parameters if the UI is hide. For example, I will move some objects in the scene using Graph UI panel, so I also want to observe full rendering results while adjusting scene parameters.
